global_settings/wagtail_hooks.py

```diff
@@ -36,12 +36,20 @@ def register_strikethrough_feature(features):
 
 @hooks.register('after_edit_page')
 def purge_cloudfront_caches(page, request):
-    try:
-        distribution = CloudfrontDistribution.objects.all()[0]
-        cf = boto.connect_cloudfront()
-        cf.create_invalidation_request(distribution.distribution_id, [])
-    except: #TODO: This is a broad exception - but we are having issues with the cache. Will tighten down once we start using it.
-        pass
+    distribution = CloudfrontDistribution.objects.all()[0]
```

Comment:
do you need to do a null check on this?

Reply:
or an empty check on `objects.all()` being `[]`?